packages/library/src/commonMain/kotlin/io/realm/internal/NotifierRealm.kt

```diff
@@ -0,0 +1,15 @@
+package io.realm.internal
+
+import io.realm.BaseRealm
+import io.realm.RealmConfiguration
+import io.realm.interop.RealmInterop
+import kotlinx.coroutines.CoroutineDispatcher
+
+/**
+ * Live Realm on which notifications can be registered.
+ */
+internal class NotifierRealm : BaseRealm {
```

Comment:
BaseRealm is currently public because it holds properties available to the user (so we cannot make it internal). Keeping it abstract at least makes sure that people don't accidentally create instances of it.

Reply:
But I agree, this class be removed